seed/energy/meter_data_processor/kairos_insert.py

```diff
@@ -9,6 +9,7 @@
 from django.conf import settings
 
 _log = logging.getLogger(__name__)
+KairosDB_Batch_Insert_Size = 1000
```

Comment:
can you make this a config variable with 1000 as the default? Have you done testing on changing this number?